app/views/form/guidance/_financial_calculations_shared_ownership.html.erb

```diff
@@ -1,12 +1,20 @@
 <%= govuk_details(summary_text: "How the financial values are calculated") do %>
   <p class="govuk-body">
-    <% if log.type == 18 %>
-      The mortgage amount <%= question_link("mortgage", log, current_user) %>,
-      cash deposit <%= question_link("deposit", log, current_user) %>,
-      and cash discount <%= question_link("cashdis", log, current_user) %>
-    <% else %>
-      The mortgage amount <%= question_link("mortgage", log, current_user) %>
-      and cash deposit <%= question_link("deposit", log, current_user) %>
+    <% if log.mortgage_used? || log.mortgageused.blank? %>
+      The mortgage amount <%= log.mortgageused.blank? ? question_link("mortgageused", log, current_user) : question_link("mortgage", log, current_user) %>
+      <% if log.type == 18 %>
+        , cash deposit <%= question_link("deposit", log, current_user) %>,
+        and cash discount <%= question_link("cashdis", log, current_user) %>
+      <% else %>
+        and cash deposit <%= question_link("deposit", log, current_user) %>
+      <% end %>
+    <% elsif log.mortgage_not_used? || log.mortgage_use_unknown? %>
+        <% if log.type == 18 %>
+          The cash deposit <%= question_link("deposit", log, current_user) %>,
+          and cash discount <%= question_link("cashdis", log, current_user) %>
+        <% else %>
+          Cash deposit <%= question_link("deposit", log, current_user) %>
+        <% end %>
     <% end %>
     added together must equal
```

Comment:
If we go down the second branch (`elsif log.mortgage_not_used? || log.mortgage_use_unknown?`) then if log type is not 18 and we go down the second branch again (`Cash deposit <%= question_link("deposit", log, current_user) %>`) ... the full sentence looks like it'll be: **Cash deposit added together must equal.....**, this looks like the one case where the phrase "added together" won't flow.

Reply:
Does it look better now?